napari/layers/shapes/_shapes_mouse_bindings.py

```diff
@@ -414,7 +414,7 @@ def polygon_creating(layer: Shapes, event: MouseEvent) -> None:
         coordinates = layer.world_to_data(event.position)
         move_active_vertex_under_cursor(layer, coordinates)
 
-        if layer._mode == Mode.ADD_POLYGON_LASSO:
+        if layer._mode in [Mode.ADD_POLYGON_LASSO, Mode.ADD_POLYLINE]:
```

Comment:
I can't suggest, but could you update the docstring to reflect that this is also for drawing paths (my preference) or polylines (current term)?

Reply:
Updated to include path.